sdks/java/io/jms/pom.xml

```diff
@@ -59,6 +46,15 @@
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-jar-plugin</artifactId>
       </plugin>
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>findbugs-maven-plugin</artifactId>
```

Comment:
OK, as said in previous comment, I used `<onlyAnalyze/>` to avoid to create a new file. Instead I will use `findbugs-filter.xml`.